stdlib/LinearAlgebra/test/bidiag.jl

```diff
@@ -297,9 +300,14 @@ Random.seed!(1)
             # test pass-through of mul! for AbstractTriangular*Bidiagonal
             Tri = UpperTriangular(diagm(1 => T.ev))
             @test Array(Tri*T) ≈ Array(Tri)*Array(T)
+            # test mul! for Diagonal*Bidiagonal
+            C = Matrix{elty}(undef, n, n)
+            Dia = Diagonal(T.dv)
+            @test mul!(C, Dia, T) ≈ Array(Dia)*Array(T)
```

Comment:
What about testing also `mul!(C,::Bidiagonal,::Diagonal)`, or is there no such method?

Reply:
There isn't one in `bidiag.jl`, at least...

Reply:
Hm, there is something related going on in #31889, maybe we should return to this once that specialized functionality exists.